spec/features/merchants/dashboard/show_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe "Merchant Dashboard Page" do
+  describe "As a merchant, when I view my dashboard" do
+
+    it "I can see the name and full address of merchant I work for" do
```

Comment:
I couldn't get a before(:each) action to work in this file - would like to look more into this during a refactor

Reply:
I was having issues with before :each as well.